### DISCUSSION TITLE
Commit 8395 does not install database

### DISCUSSION THREAD

**Discussion by efeoli** [timschofield/webERP#204] [Installation Issues] [answered]
Hi,
I was waiting for your installer rewritted script, since the application could not be installed when clicking the INSTALL button.

Despite the improvements the files get installed but not  the database.

Enrique

**Comment by timschofield** [comment]
Hello Enrique,

The changes committed yesterday were just some minor fixes that happened in a general sweep of the code. The new installer will be loaded up later today I hope. I will let you know when, and it would be great if you could test again.

Tim

**Comment by timschofield** [comment]
The new installer is now in the master branch. I would really appreciate it if you could test on your system.

Thanks
Tim

**Comment by efeoli** [comment]
Hello Tim.
Thank you for letting me know about the latest master.

It gets to the database credentials, but it will accept the data. Tried repeatedly.

Attachments might help.
Enrique 



![Captura de pantalla_25-10-2024_103115_kwa bioacyl com after](https://github.com/user-attachments/assets/7fc7e02d-d16a-4860-bdf1-56a707f34609)
![Captura de pantalla_25-10-2024_102628_kwa bioacyl com](https://github.com/user-attachments/assets/5f771738-4e32-4fa3-a230-1c30e6247947)

**Comment by efeoli** [comment]
I clarify: it will not accept data

**Comment by efeoli** [comment]
Also images are diproportionatelly large and disturbs format for data to be eneterd and it is not what goes where.

**Comment by timschofield** [comment]
It looks like the css isn't applying. Can you try ctrl-f5 in the browser to force a css reload.

Tim

**Comment by efeoli** [comment]
Yes it did loas the correct css and the procedure went through, but it loaded only 57 tables of the database ínstead of 175.
Enrique

**Comment by efeoli** [comment]
Not it will perform upgrade to a 4.15.2 database. It gets stuck at steps 5,6,7.
![Screenshot 2024-10-25 at 12-23-51 Database Upgrade](https://github.com/user-attachments/assets/f510e2d0-7192-4f5c-9295-820be87e14fd)

**Comment by timschofield** [comment]
What was the last table it created? It creates them in alphabetic order

**Comment by efeoli** [comment]
These are the 57 tables that it creatres.
![Captura de pantalla_25-10-2024_13748_da4613 squidix net](https://github.com/user-attachments/assets/7f0f1ddb-a0f2-4148-ab10-f3b0a980d13b)

**Comment by timschofield** [comment]
I have just pushed two files
install/tables/jnltmpldetails.sql
install/tables/jnltmplheader.sql

Can you try downloading them and trying again?

Tim

**Comment by efeoli** [comment]
It only improved installing 60 tablea and the last one is “modules”.

Enrique

**Comment by efeoli** [comment]
I also the notice of the 5,6,7 steps to go attached above.

**Comment by timschofield** [comment]
That's very strange as the 60th table is labelfields.

Did you get any error message?

**Comment by efeoli** [comment]
No error message.

Would you like a snapshot of tables?

**Comment by timschofield** [comment, marked as answer]
Yes please.
I ran the installer with mysql mode set as STRICT_TRANS_TABLES and it threw a couple of errors that I have corrected and pushed.

Are you deleting the database between attempts?

**Comment by efeoli** [comment]
I was emptying the database. I will delete and rey again.

We are 6 hours earlier in Costa Rica. I will be available tomorrow in case you will need further testimg.
![Captura de pantalla_25-10-2024_153658_da4613 squidix net](https://github.com/user-attachments/assets/bd62d9af-54e7-44e8-a406-88d1563b0928)

**Comment by efeoli** [comment]
Success!

I comment tomorrow on how it is working.

**Comment by timschofield** [comment]
Glad it now works for you.

I ran the installer again with everything (web server/mysql/PHP) set on their most severe settings. Found 3 more errors that I have now fixed and uploaded

**Reply by efeoli** [reply]
Hello Tim,
If useful for you I am listing details of what I found exploring the latest commit.

The version that gets installed is 4.15.1 and not 4.15.2

The very useful tab for”WO Items ready to produce” under manufacturing, is not present.

With latest fixes to the installer the database upgrade screen shows up for importing data from prior version, but it gets stuck at steps 5,6,7 as per screenshot above.

Reports are uploaded but not shown in the main menu.

Otherwise,  everthing runs fine under PHP 8.3, which is a tremedous advancement and the many improvements are excellent. I wonder if what pointed out may be improved?

Enrique

**Comment by timschofield** [comment]
It is very useful. I am a bit confused that you are getting the database
upgrade screen. Someone else got that. Can you look in your config table.
You should have a record where confname='DBUpdateNumber'. If you do, then
what is the value of confvalue field? It should be 8

Thanks
Tim

On Sat, 26 Oct 2024 at 16:39, efeoli ***@***.***> wrote:

> Hello Tim,
> If useful for you I am listing details of what I found exploring the
> latest commit.
>
> The version that gets installed is 4.15.1 and not 4.15.2
>
> The very useful tab for”WO Items ready to produce” under manufacturing, is
> not present.
>
> With latest fixes to the installer the database upgrade screen shows up
> for importing data from prior version, but it gets stuck at steps 5,6,7 as
> per screenshot above.
>
> Reports are uploaded but not shown in the main menu.
>
> Otherwise, everthing runs fine under PHP 8.3, which is a tremedous
> advancement and the many improvements are excellent. I wonder if what
> pointed out may be improved?
>
> Enrique
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/204#discussioncomment-11061395>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LHWFS2SEEELPAWW3ADZ5PATZAVCNFSM6AAAAABQU6JYICVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBWGEZTSNI>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***
> com>
>


-- 
www.weberpafrica.com
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Reply by efeoli** [reply]
For the last commit installation is 8, for the attempt to import older data base iformation I would need to try again. I will do so if that is what you nerdo to know.

**Comment by timschofield** [comment]
Ah, it should be 7

Tim

On Sat, 26 Oct 2024 at 17:15, Tim Schofield ***@***.***>
wrote:

> It is very useful. I am a bit confused that you are getting the database
> upgrade screen. Someone else got that. Can you look in your config table.
> You should have a record where confname='DBUpdateNumber'. If you do, then
> what is the value of confvalue field? It should be 8
>
> Thanks
> Tim
>
> On Sat, 26 Oct 2024 at 16:39, efeoli ***@***.***> wrote:
>
>> Hello Tim,
>> If useful for you I am listing details of what I found exploring the
>> latest commit.
>>
>> The version that gets installed is 4.15.1 and not 4.15.2
>>
>> The very useful tab for”WO Items ready to produce” under manufacturing,
>> is not present.
>>
>> With latest fixes to the installer the database upgrade screen shows up
>> for importing data from prior version, but it gets stuck at steps 5,6,7 as
>> per screenshot above.
>>
>> Reports are uploaded but not shown in the main menu.
>>
>> Otherwise, everthing runs fine under PHP 8.3, which is a tremedous
>> advancement and the many improvements are excellent. I wonder if what
>> pointed out may be improved?
>>
>> Enrique
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/timschofield/webERP/discussions/204#discussioncomment-11061395>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAAL6LHWFS2SEEELPAWW3ADZ5PATZAVCNFSM6AAAAABQU6JYICVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBWGEZTSNI>
>> .
>> You are receiving this because you modified the open/close state.Message
>> ID: ***@***.***
>> com>
>>
>
>
> --
> www.weberpafrica.com
> @TimSchofield2
> Blog: https://kwamoja.home.blog/
>


-- 
www.weberpafrica.com
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Reply by efeoli** [reply]
Should I change to 7 and try to import the prior data?

**Reply by efeoli** [reply]
What happens is the update screen gets activated, but this time the upgrade runs satisfactorilly and completes 9 upgrades. However the versión is still 4.15.1 and the WO ready to produce tab still is absent.

**Comment by timschofield** [comment]
Try changing it to 7 see what happens then.

Tim

On Sat, 26 Oct 2024 at 17:43, efeoli ***@***.***> wrote:
>
> Should I change to 7 and try to import the prior data?
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you modified the open/close state.Message ID: ***@***.***>



-- 
www.weberpafrica.com
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by timschofield** [comment]
Don't worry about the version number, I will update that closer to release
date. I am confused about the WO ready to produce functionality you
mention. Was this on older versions?

On Sat, 26 Oct 2024 at 18:38, efeoli ***@***.***> wrote:

> What happens is the update screen gets activated, but this time the
> upgrade runs satisfactorilly and completes 9 upgrades. However the versión
> is still 4.15.1 and the WO ready to produce tab still is absent.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/204#discussioncomment-11062106>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LC5C2OUMZGAZ4KITATZ5POT5AVCNFSM6AAAAABQU6JYICVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBWGIYTANQ>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***
> com>
>


-- 
www.weberpafrica.com
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Reply by efeoli** [reply]
Yes quite useful just above MRP tab, helps complete materials inventory in order to proceed with production of work order.

**Comment by timschofield** [comment]
Found it, have uploaded install/demo.sql install/sql/menuitems.sql files

On Sat, 26 Oct 2024 at 19:26, efeoli ***@***.***> wrote:

> Yes quite useful just above MRP tab, helps complete materials inventory in
> order to proceed with production of work order.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/204#discussioncomment-11062378>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LGGHQ3SAPUFZKULHLLZ5PUH7AVCNFSM6AAAAABQU6JYICVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBWGIZTOOA>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***
> com>
>


-- 
www.weberpafrica.com
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Reply by efeoli** [reply]
Thank you for it is a great resource.  I will install and ley you know.

**Reply by efeoli** [reply]
The installtion goes through but I get a logged out message as attached, when legging in.

**Reply by efeoli** [reply]
![Screenshot 2024-10-26 at 15-22-48 ](https://github.com/user-attachments/assets/1a2708fd-35d8-45a5-9b07-8a05c1e77548)

**Reply by efeoli** [reply]
Wait. after clearing cachets it was possible to login however main menu does not respond while selectos ítems do work well.

**Comment by timschofield** [comment]
Hi, it's late here now, I will look at this in the morning

Thanks
Tim

On Sat, 26 Oct 2024 at 21:39, efeoli ***@***.***> wrote:

> Screenshot.2024-10-26.at.15-22-48.png (view on web)
> <https://github.com/user-attachments/assets/1a2708fd-35d8-45a5-9b07-8a05c1e77548>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/204#discussioncomment-11063184>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LGOTAGUVNJAMRK2OJ3Z5QDZFAVCNFSM6AAAAABQU6JYICVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMBWGMYTQNA>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***
> com>
>


-- 
www.weberpafrica.com
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Reply by efeoli** [reply]
Hello Tim,
In commit 8415 the installer does not configure the user, so you can’t get beyond the login screen.

If you configure the user manually in the database, you can log in but permits is complicated to set, so that one is not able to navigate the appiclation.

Also the Tab for WO ready to produce does not show. Would you agree that it is a must for the upcoming v5.0? I have been using webERP for more than a decade and am elated that it will evolve to a sophisticated state and please consider the feedback as a honest effort to help.

Enrique

**Reply by efeoli** [reply]
Hello Tim,
here is further feedback attached on your recent webERP commit.
Enrique 

[webERP commit 8419.pdf](https://github.com/user-attachments/files/17621442/webERP.commit.8419.pdf)
